composer.json

```diff
@@ -21,7 +21,7 @@
         "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
         "symfony/config": "~4.4 || ^5.0 || ^6.0",
         "symfony/http-kernel": "~4.4 || ^5.0 || ^6.0",
-        "monolog/monolog": "~1.22 || ~2.0"
+        "monolog/monolog": "~1.22 || ~2.1"
```

Comment:
This package already allow `monolog/monolog 2.1, this change won't fix your issue.

> ~2.0 is equivalent to >=2.0 <3.0.0

From composer documentation : https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

Reply:
That's true ! sorry about that, I think this requires more investigation as the issue is related to other packages, this is the output of the composer:

```
  Problem 1
    - monolog/monolog[1.22.0, ..., 1.26.1] require psr/log ~1.0 -> found psr/log[1.0.0, ..., 1.1.4] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - monolog/monolog[2.0.0, ..., 2.0.2] require php ^7.2 -> your php version (8.0.8) does not satisfy that requirement.
    - monolog/monolog[2.1.0, ..., 2.3.2] require psr/log ^1.0.1 -> found psr/log[1.0.1, ..., 1.1.4] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - symfony/monolog-bundle v3.7.0 requires monolog/monolog ~1.22 || ~2.0 -> satisfiable by monolog/monolog[1.22.0, ..., 1.26.1, 2.0.0, ..., 2.3.2].
    - Root composer.json requires symfony/monolog-bundle ^3.7 -> satisfiable by symfony/monolog-bundle[v3.7.0].
```

I think it's better to open a new issue regarding the support of PHP 8.0.

I'll close this PR.

Reply:
PHP 8 support is not your problem. You need to downgrade `psr/log` in order to install Monolog.

Reply:
See also Seldaek/monolog#1564

Reply:
@derrabus thank you, I downgraded the `psr/log` to 1.1.4 and it works !